person.rb

```diff
@@ -15,6 +18,10 @@ def can_use_services?
     true
   end
 
+  def correct_name
+    @name
+  end
+
   private
 
   def of_age?
```

Comment:
Optional 

The if block isn't necessary `@age >= 18` would return a boolean value.